src/Services/Versions.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Avtocod\Specifications\Services;
+
+use Composer\InstalledVersions;
+
+/**
+ * Wrapper for Composer runtime utility `InstalledVersions`.
+ */
+final class Versions
```

Comment:
Where are the tests?

Reply:
Tests added https://github.com/avtocod/specs-php/pull/16/commits/7325fe04296007d0d406cff75b0a2b2e31289669